client/src/themes/theme.js

```diff
@@ -4,19 +4,26 @@ export const theme = createMuiTheme({
   typography: {
     fontFamily: '"Roboto"',
     h1: {
-      fontSize: '1.5em',
+      fontSize: '1.7em',
       fontWeight: 'semi-bold',
       textAlign: 'center'
     },
     h2: {
-      fontSize: '2.75em',
-      fontWeight: 'semi-bold',
+      fontSize: '2.2em',
+      fontWeight: '500',
       textAlign: 'center'
     },
     h3: {
-      fontSize: '2em',
+      fontSize: '1.9em',
       fontWeight: 'semi-bold'
```

Comment:
I added some padding for h3 in my LaunchProjects branch. Is that okay? Otherwise I think this looks good